### ISSUE TITLE
cas particulier élément d'interface inactif sur test contraste

### ISSUE THREAD

**Issue by goetsu** [DISIC/rgaa_referentiel#11]
le RGAA ne semble pas prendre en compte le cas suivant présent sur le critère 1.4.3 des WCAG : 
"Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement."
Il faudrait donc ajouter cela dans le cas particulier des critères 3.3 et 3.4

**Comment by access42** [comment]
Merci beaucoup de votre retour.

Une issue a été ouverte et sera traitée lors de la prochaine mise à jour.
